File: src/tool/spring-boot-generator/output/railway/db/damageRepositoryPosition.js
let damageRepositoryPosition = {
	name: "damageRepositoryPosition",
	description: "伤损归档-受伤位置扩展",
	columns: {
		bridgeTunnel: {
			name: "bridgeTunnel",
			description: "桥隧道口",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		damagePosition: {
			name: "damagePosition",
			description: "损伤位置",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		direction: {
			name: "direction",
			description: "行别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		holePosition: {
			name: "holePosition",
			description: "孔裂位置",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		lineCategory: {
			name: "lineCategory",
			description: "线别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		lineId: {
			name: "lineId",
			description: "线路编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		lineName: {
			name: "lineName",
			description: "线名",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		miles: {
			name: "miles",
			description: "里程",
			nullable: true,
			type: {
				name: "float"
			}
		},
		orderDetailId: {
			name: "orderDetailId",
			description: "记录编号",
			isPk: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		stationId: {
			name: "stationId",
			description: "车站编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		stockExchange: {
			name: "stockExchange",
			description: "股别",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		thighTrackId: {
			name: "thighTrackId",
			description: "股道号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		}
	}
};

let columnsArray = [
	"bridgeTunnel",
	"createTime",
	"damagePosition",
	"direction",
	"holePosition",
	"lineCategory",
	"lineId",
	"lineName",
	"miles",
	"orderDetailId",
	"stationId",
	"stockExchange",
	"thighTrackId",
	"updateTime"
];

function create() {
  return {};
}

damageRepositoryPosition.columnsArray=columnsArray;
damageRepositoryPosition.create=create;
module.exports=damageRepositoryPosition;

File: src/tool/spring-boot-generator/output/railway/railway/src/main/java/com/railway/pojo/entity/DamageRepositoryPosition.java
package com.railway.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("伤损归档-受伤位置扩展")
@Getter
@Setter
public class DamageRepositoryPosition  {

    @ApiModelProperty("桥隧道口")
    private Integer bridgeTunnel;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("损伤位置")
    private Integer damagePosition;

    @ApiModelProperty("行别")
    private Integer direction;

    @ApiModelProperty("孔裂位置")
    private Integer holePosition;

    @ApiModelProperty("线别")
    private Integer lineCategory;

    @ApiModelProperty("线路编号")
    private Integer lineId;

    @ApiModelProperty("线名")
    private String lineName;

    @ApiModelProperty("里程")
    private Float miles;

    @ApiModelProperty("记录编号")
    private Integer orderDetailId;

    @ApiModelProperty("车站编号")
    private Integer stationId;

    @ApiModelProperty("股别")
    private Integer stockExchange;

    @ApiModelProperty("股道号")
    private Integer thighTrackId;

    @ApiModelProperty("更新时间")
    private Date updateTime;

}
